play-json/shared/src/main/scala/Json.scala

```diff
@@ -413,6 +418,31 @@ object Json extends JsonFacade {
    */
   sealed trait MacroOptions
 
+  object MacroOptions {
+    /**
+     * Defines the default macro options if no type is supplied.
+     *
+     * Since you can't have defaults for a type parameter (unless it's a contravariant type, it will default to
+     * Nothing), we supply the default via an implicit parameter.
+     */
+    trait Default[O <: Json.MacroOptions]
```

Comment:
`Fallback` ?

Reply:
I think the naming is going to be confusing no matter what, since the object itself is meaningless and just used for its type. Maybe I should add a link to https://stackoverflow.com/a/29415519?

Reply:
No strong opinion. Fallback could also denote that it is something use only if no explicit options are defined